pytools/tag.py

```diff
@@ -36,7 +36,9 @@
 
 Tag Interface
 ---------------
+.. <!--``normalize_tags`` undocumented for now. (Not ready to commit.)-->
```

Comment:
Out of curiosity: How come this got wrapped in an HTML comment?

Reply:
This was just to prevent this from being rendered in the generated HTML while keeping this here as a comment

Reply:
I think 57678bf suffices.